aiidalab_launch/instance.py

```diff
@@ -286,34 +286,45 @@ async def echo_logs(self) -> None:
     @staticmethod
     async def _init_scripts_finished(container: Container) -> None:
         loop = asyncio.get_event_loop()
-        LOGGER.debug("Waiting for init services to finish...")
+        LOGGER.info("Waiting for init services to finish...")
```

Comment:
I changed a few imo useful debug statements into info. Main reason is that I find that '-vv' does not print enough, while '-vvv' prints way too much, mostly due to notebook polling. Happy to reverse this though.